Redesigned DualShock 4 Theme (again) & Reorganized DS4 Theme files

AL2009man opened the pull request:
## Redesigned DualShock 4 Theme, again.

Previously, the current DS4 Theme was taken directly from the [DualShock 4 Button Icons Mod in Devil May Cry 5](https://steamcommunity.com/sharedfiles/filedetails/?id=1679480972) as it was the highest quality image I could find at the time. 

Months later, I was gifted an extremely raw DS4 Lineup art. This gave me a good reason to revisit the theme once again.

After cleaning up (and figuring out how to separate the Triggers again, this one was a doozey and prevented me from making everything happened.), everything's now powered by SVG-to-PNG. The look and feel of the theme should look much smoother and less aliased with an added benefit of modular customization and futureproofing.

![Redesigned DS4 Theme image size comparison](https://user-images.githubusercontent.com/67606569/123736670-8d635980-d86f-11eb-9003-02d44453e04e.png)

*Note: The image above does not represent the actual theme's base Overlay Image size.

There's a lot of changes I've made with the redesigned theme. One of the biggest changes is that the Image Size (and Canvas Image) now matches the DualSense Theme (previously 1207x793), it should be more consistent with the rest of the DS4 Default themes. All of this is backported over to the DMC5 Theme.

I also gave the V2 Button/Trigger/Touchpad Colors to the default DS4 V2 Theme, while changing the Front Lightbar for the V2 Theme)

![image](https://user-images.githubusercontent.com/67606569/123731511-402eba00-d866-11eb-9ce4-f4c8dce1b910.png)


## Cleaned up [dualshock4] folder

There were two specific issues I was having with the folder:

1. Using Github Web to update the image wasn't going thru.
2. the DS4 Theme Folder is bloated.

Since then, I switched to Github Desktop and I find it easier to make changes in a rapid sense without having to manually remove everything (initially, I took the nuclear option and deleted the DS4 Theme Folder). 

## Others:

- Adjusted Trigger and Analog Stick placement.
- Shorted Devil May Cry 5 Theme name from [Devil May Cry 5 Controller Overlay] to [Devil May Cry 5].
- Adjusted the size of the Trailpad to compensate for the Larger Image size
- Added Paddle, Bumpers, Menu and Misc Buttons to Flee Variables on README.md.
- Updated Controller Icons for Generic and Unknown Controller.
- [About VSCView Menu] While the tabs hasn't been updated, I updated the Licensed tag.

Nielk1 reviewed (changes requested):
This is actually incorrect, the section lists the elements of each Control part rather than the specific Control part names.  Another section could be added to list items such as `quad_left` and `quad_right`, but the intent here was to describe that a `ControlButtonQuad` has sub-variables `s`, `e`, `w`, and `n`.

Edit: It doesn't look like this noted the file I was looking at when I said this.  README.MD

AL2009man commented:
> This is actually incorrect, the section lists the elements of each Control part rather than the specific Control part names. Another section could be added to list items such as `quad_left` and `quad_right`, but the intent here was to describe that a `ControlButtonQuad` has sub-variables `s`, `e`, `w`, and `n`.
> 
> Edit: It doesn't look like this noted the file I was looking at when I said this. README.MD

ok, I reverted the naming scheme. 

However, the [ControlButtonPair] naming scheme will have to be kept since `grip` and `menu` weren't listed in the Flee Variable, reverted those back may lead into confusion for Theme Makers in the future. Best to keep that temporary until we figured out a much better name for it.
